tests/test_cli.py

```diff
@@ -47,4 +47,4 @@ def test_startproject():
 
         assert result.exit_code == 0
         assert os.listdir() == [project_name]
-        assert os.listdir(project_name) == project_files
+        assert os.listdir(project_name).sort() == project_files.sort()
```

Comment:
`.sort()` returns `None`, so you are comparing `None == None`. 😉 

This should be:

```python
assert sorted(os.listdir(project_name)) == sorted(project_files)
```

Reply:
done